<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_client</class>
 <widget class="QDialog" name="edit_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back2.png);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>651</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Verification">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_enter_id">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_confirm_verification">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Edit">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);
</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Last name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>130</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>180</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>131</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_edit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>350</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name_edit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lastname_edit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_number_edit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_email_edit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>100</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_type_edit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>180</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_age_edit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>260</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);
</string>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_edit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 127);
</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
